native_lib/DDGSyncCrypto.h

```diff
@@ -1,6 +1,10 @@
+#ifdef _WIN32
+#include <sodium.h>
```

Comment:
Is this necessary? The other 3 includes (and a whole bunch more) are included by `<sodium.h>`.

Reply:
Sadly yes, I wasn't able to figure out how to make the `#import ...` working. I even managed to switch from `clang-cl` to `msvc`, but still :/

Fwiw, `#include <sodium.h>` is the [official recommendation](https://libsodium.gitbook.io/doc/quickstart):

> A project using libsodium should include the sodium.h header. Including individual headers is neither required nor recommended.

Reply:
That's nice to know, thanks. Does adding `#pragma once` to the top of the header file achieves the same?

Reply:
It does, but obviously relies on the header author writing it (and it's worth noting that `#pragma once` is also not standard C++, though it's somewhat universally supported by compilers).